views/includes/metaHead.njk

```diff
@@ -3,7 +3,7 @@
 <link rel="stylesheet" type="text/css" href="{{ 'stylesheets/style.css' | getCachebustedPath }}">
 <link href="https://fonts.googleapis.com/css?family=Poppins:300,500,600|Roboto:400,700" rel="stylesheet">
 
-<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=HTMLPictureElement,Promise,Array.prototype.find"></script>
+<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=HTMLPictureElement,Promise,Array.prototype.find,Element.prototype.closest"></script>
```

Comment:
I've added `Element.prototype.closest` but all but IE11 supports it, and it's unlikely IE11 users would trigger this so we might be able to get away without this.